### ISSUE TITLE
Error: multiple-value uuid.NewV4() in single-value context

### ISSUE THREAD

**Issue by 0xdevalias** [OJ/gobuster#68]
While building:
```
./main.go:535: multiple-value uuid.NewV4() in single-value context
./main.go:560: multiple-value uuid.NewV4() in single-value context
```

Code refs:
* https://github.com/OJ/gobuster/blob/master/main.go#L535
* https://github.com/OJ/gobuster/blob/master/main.go#L560

Library causing the issue:
* https://github.com/satori/go.uuid

Looks like the usage has changed to supply an error value in the return as well, eg:

```
u, err := uuid.NewV4()
```

This seems to be the change that caused it:
* https://github.com/satori/go.uuid/commit/0ef6afb2f6cdd6cdaeee3885a95099c63f18fc8c

If the dependencies were pinned to known working versions (#61), errors like this shouldn't come up.

**Comment by 0xdevalias** [comment]
I'm hesitant to make any PR's given the mass of changes waiting to land in [1.4-dev](https://github.com/OJ/gobuster/tree/1.4-dev)

**Comment by BBerastegui** [comment]
Just for the sake of people coming here looking for this error (like me):

`sed -i 's/guid := uuid.NewV4()/guid, _ := uuid.NewV4()/g' gobuster/main.go`

That should do the trick. In a really ugly way.

**Comment by iddoeldor91** [comment]
@BBerastegui just in time, thank you

**Comment by OJ** [comment]
This has been fixed by @eur0pa, and landed in 1.4 now.
